save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkContestProjectController.kt

```diff
@@ -144,6 +144,12 @@ class LnkContestProjectController(
         Mono.justOrEmpty<Project>(projectService.findByNameAndOrganizationName(projectName, organizationName)),
```

Comment:
`findWithPermission` and `Permission.READ`?

Reply:
Ok, done